presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusColumn.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.prestosql.spi.type.Type;
+
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static java.util.Objects.requireNonNull;
+
+public final class PrometheusColumn
```

Comment:
Maybe use `PrometheusColumnHandle` instead?

Reply:
So like replace all usages of `PrometheusColumn` with `PrometheusColumnHandle`? Is there a big benefit?

Reply:
> Is there a big benefit?

Is there a big benefit of having these two classes? How do they differ?

Reply:
There are a number of different usages and diffs. `toString()` is substantially different and I'm not sure an overload is possible or desirable there. And I think the separate class is easier for usage with `PrometheusTable`.